rhel-7.template

```diff
@@ -20,6 +20,7 @@ part / --fstype="ext4" --size=10240
 repo --name=base --baseurl=${base_repo_url}
 repo --name=updates --baseurl=${updates_repo_url}
 repo --name=cdk --baseurl=${cdk_repo_url}
+repo --name=hvkvp --baseurl=${hvkvp_repo_url}
```

Comment:
@gbraad you just remove this line because we will have `go-hvkvp` package as part of cdk repo itself.